src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerCorporateValidator.java

```diff
@@ -59,16 +61,13 @@ public Errors validate(List<BeneficialOwnerCorporateDto> beneficialOwnerCorporat
 
             validateStartDate(beneficialOwnerCorporateDto.getStartDate(), errors, loggingContext);
 
-            List<NatureOfControlType> fields = new ArrayList<>();
-            if (Objects.nonNull(beneficialOwnerCorporateDto.getBeneficialOwnerNatureOfControlTypes())) {
-                fields.addAll(beneficialOwnerCorporateDto.getBeneficialOwnerNatureOfControlTypes());
-            }
-            if (Objects.nonNull(beneficialOwnerCorporateDto.getNonLegalFirmMembersNatureOfControlTypes())) {
-                fields.addAll(beneficialOwnerCorporateDto.getNonLegalFirmMembersNatureOfControlTypes());
-            }
-            if (Objects.nonNull(beneficialOwnerCorporateDto.getTrusteesNatureOfControlTypes())) {
-                fields.addAll(beneficialOwnerCorporateDto.getTrusteesNatureOfControlTypes());
-            }
+            var fields = Stream.of(
```

Comment:
Do we have good unit test coverage for this? It looks good but want everything to be low risk.

Reply:
Just ran the tests with coverage and looks like we do 

![image](https://github.com/companieshouse/overseas-entities-api/assets/124157110/0f947958-add9-4685-b807-af745db1fb11)

Give me a moment and I'll look to see if it's good enough though

Reply:
Yeah looks like they're quite heavily tested with 
- testErrorReportedWhenNoNatureOfControlValuesAreAllNull
- testErrorReportedWhenNoNatureOfControlValuesAreAllEmpty
- testNoErrorReportedWhenSomeNatureOfControlValuesAreAllEmptyAndNull
- testErrorReportedWhenIsOnPublicRegisterNameFieldIsNull